pandas/tests/arithmetic/test_numeric.py

```diff
@@ -74,33 +75,22 @@ def test_compare_invalid(self):
 
 # ------------------------------------------------------------------
 # Numeric dtypes Arithmetic with Datetime/Timedelta Scalar
+index_or_series_params = [
+    pd.Series,
+    pd.Index,
+]  # type: List[Union[Type[pd.Index], Type[pd.RangeIndex], Type[pd.Series]]]
+left = [pd.RangeIndex(10, 40, 10)]  # type: List[Union[Index, Series]]
+for cls in index_or_series_params:
```

Comment:
isn’t this any_numeric_dtype ?

Reply:
This seems to include float16. Aside from that, they look the same at a glance.

Reply:
we barely support float16 would just remove and use the fixture